<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyServerMain</class>
 <widget class="QMainWindow" name="MyServerMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyServer_app</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>TCP server</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="3">
        <widget class="QPushButton" name="pbStartServer">
         <property name="text">
          <string>Start server</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="sbPort">
         <property name="minimum">
          <number>1000</number>
         </property>
         <property name="maximum">
          <number>60000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lPort">
         <property name="text">
          <string>port</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="pbStopServer">
         <property name="text">
          <string>Stop server</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>onnx model</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="2">
        <widget class="QPushButton" name="pbReadModel">
         <property name="text">
          <string>read</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="tbReadModel">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="leReadModel">
         <property name="text">
          <string>../../model/MyRadarNet_best.onnx</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>filepath</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QListWidget" name="lwServer"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
